### PR TITLE
Built-in certificate behavior

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/w3c/webrtc-pc/issues/1322

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -1687,12 +1687,14 @@
...
                 any local candidates that have been gathered by the agent.</p>
-                <p>The value for <code>certificates</code> in the
-                <code><a>RTCConfiguration</a></code> for the
-                <code>RTCPeerConnection</code> is used to produce a set of
-                certificate fingerprints. These certificate fingerprints are
-                used in the construction of SDP and as input to requests for
-                identity assertions.</p>
+                <p>The value of
+                <code><a>RTCConfiguration</a>.certificates</code> for the
+                <code>RTCPeerConnection</code>, along with the default
```
Comment (taylor-b) [inline webrtc.html]: nit: I'd say "any default certificate(s)" since there may be 0 to many.

**Comment by aboba** [comment]
@taylor-b Are you referring to PR https://github.com/w3c/webrtc-pc/pull/1233 as the "other place"?

**Comment by fippo** [comment]
i need to redo my PR anyway so go ahead

**Comment by taylor-b** [comment]
@aboba I was referring to https://github.com/w3c/webrtc-pc/pull/1220. Both the `createOffer` and `createAnswer` now talk about certificate fingerprints

**Comment by aboba** [comment]
@taylor-b I believe this is fixed now.

**Comment by stefhak** [comment]
LGTM.

**Pull request merged**